### PR TITLE
fly.io stuff for demo.

### PR DESCRIPTION
**Pull request by ildyria**
Files necessary for the demo available here: https://lychee-demo.fly.dev/

### CODE REVIEW

**Review by qwerty287 — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        ln -sf /etc/nginx/sites-available/default-octane /etc/nginx/sites-enabled/default; \
+    fi
+
+# Multi-stage build: Build static assets
+# This allows us to not include Node within the final container
+FROM node:${NODE_VERSION} as node_modules_go_brrr
+
+RUN mkdir /app
+
+RUN mkdir -p  /app
```
Comment (d7415) [inline Dockerfile]:
```suggestion
```

This is repeated above

**Comment by kamil4** [comment]
Wouldn't it make more sense for the demo site to be in its own repo, rather than in the main development repo? I guess it's not a big deal if it's not part of the official release tarballs, but still...

**Comment by ildyria** [comment]
No strong opinion.

**Comment by d7415** [comment]
> Wouldn't it make more sense for the demo site to be in its own repo, rather than in the main development repo? I guess it's not a big deal if it's not part of the official release tarballs, but still...

That would make sense. Would save putting the images in the main git repo too.

**Pull request closed without merging**